pkg/bigquery/sanitize_test.go

```diff
@@ -0,0 +1,319 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bigquery
+
+import (
+	"strings"
+	"testing"
+)
+
+func genStringLen(num int, char rune) string {
+	if num < 0 {
+		panic("cannot make negative length string")
+	}
+	var sb strings.Builder
+	sb.Grow(num)
+	for i := 0; i < num; i++ {
+		sb.WriteRune(char)
+	}
+	return sb.String()
+}
+
+func Test_ValidateGCPProjectID(t *testing.T) {
```

Comment:
Conventionally there's not an underscore in a test name except when testing a method. https://github.com/abcxyz/readability/wiki/Go-Best-Practices#testing

Reply:
I always forget all the parts I need to change from what my IDE generates, thanks!